CONTRIBUTING.md

```diff
@@ -59,35 +59,51 @@ def count_truthy(elements: List[Any]) -> int:
 
 ### JavaScript Development
 
-First, clone this repo. Then install dependencies (`npm i`). Now you're ready to begin development.
+First clone this repo, then download and install dependencies:
 
-If you want to use a particular dataset, you'll first need to place it in `/public/logs/nodes.json`. Otherwise, you can serve randomly-generated data.
-
-Kedro-Viz uses an environment variable to configure the data source. You can set it when starting up the dev server, e.g. `DATA=random npm start` will serve random procedurally-generated data (refreshed on each page-load). This is usually the most useful setting for local development.
+```bash
+npm install
+```
 
-In other words, to run the app in development mode on a local server, use one of the following:
+Now you're ready to begin development. Start the development server:
 
-- `DATA=random npm start` --> Serve randomly-generated data
-- `DATA=animals npm start` --> Serve example test data, from `/src/utils/data.mock.js`
-- `DATA=lorem npm start` --> Serve example test data, from `/src/utils/data.mock.js`
-- `npm start` --> Serve data loaded from `/public/logs/nodes.json`
+```bash
+npm start
+```
 
 This will serve the app at [localhost:4141](http://localhost:4141/), and watch files in `/src` for changes. It will also update the `/lib` directory, which contains a Babel-compiled copy of the source. This directory is exported to `npm`, and is used when importing as a React component into another application. It is updated automatically on save in case you need to test/debug it locally (e.g. with `npm link`). You can also update it manually, by running
 
 ```bash
 npm run lib
 ```
 
+#### Data sources
+
+Kedro-Viz uses unique identifiers to configure the data source. You can configure the data source by appending a query string to the URL, e.g. `http://localhost:4141/?data=random`. You can also set it with an environment variable when starting up the dev server, e.g. `DATA=random npm start`.
```

Comment:
```suggestion
Kedro-Viz uses unique identifiers to configure the data source. You can configure the data source by appending a query string to the URL, e.g. `http://localhost:4141/?data=random`. Alternatively, you can set it by configuring an environment variable that is used when the dev server starts up: 

```bash
DATA=random npm start
```

Reply:
My suggestion is a bit passive tense-y. You could also say "Alternatively, you can set it by configurating an environment variable that the dev server uses when it starts up:"

Reply:
To me, this is less clear than the original phrasing: What I am trying to communicate is that you should set the environment variable _when you are starting the dev server_, rather than just that the environment variable is used when the server starts up, which is less specific.

How about something like this? 
> Alternatively, you can set it by configuring an environment variable when starting up the dev server: 

Also, I noticed when re-reading this paragraph that the word `configures` is repeated quite a lot. So maybe we could replace one or two of the uses of that word with a synonym? How about this:

> Kedro-Viz uses a unique identifier to determine the data source. You can configure this by appending a query string to the URL, e.g. `http://localhost:4141/?data=random`. Alternatively, you can set it with an environment variable when starting up the dev server:

Reply:
Ah, I see. OK, yes that makes sense. I like your final suggestion.